### DISCUSSION TITLE
Add "primary" as an option for the color attribute for badges.

### DISCUSSION THREAD

**Discussion by kyledoesdev** [livewire/flux#912] [Feature Requests]
It would be cool to have whatever the primary color is from your theme available as an option for the color attribute for badges. There may be a way to do this already by putting a class of text-primary or bg-primary on the badge? I am unsure, I don't think I saw anything specific to this in the docs. It would be a cool nice to have.

**Comment by jeffchown** [comment]
@kyledoesdev If you've setup your tailwind.config.js per the Theming docs (https://fluxui.dev/docs/theming) - Have you tried adding `class="bg-accent text-accent-foreground"`?  You may need to add `!` to each to override the `flux:badge` default classes.

**Reply by kyledoesdev** [reply]
I'll try that out. Thanks man

**Reply by kyledoesdev** [reply]
I think this works but it won't work if you have a badge on a navigation item. You can set badge-color as an attribute but it won't pick up on the the theming colors.

**Reply by jeffchown** [reply]
As there are subtleties to the `flux:badge` colours' CSS, e.g.:

`'red' => 'text-red-700 dark:text-red-200 bg-red-400/20 dark:bg-red-400/40'`,

maybe one solution would be for @calebporzio @joshhanley to add an `accent` colour choice for `flux:badge`.

**Reply by kyledoesdev** [reply]
> As there are subtleties to the `flux:badge` colours' CSS, e.g.:
> 
> `'red' => 'text-red-700 dark:text-red-200 bg-red-400/20 dark:bg-red-400/40'`,
> 
> meybe one solution would be for @calebporzio @joshhanley to add an `accent` colour choice for `flux:badge`.

This would be an awesome addition!

**Comment by LuciusAB** [comment]
This would truly be a great addition to badges, to make them fit the theme colors if you don't use a standard tailwind color as accent.
